### DISCUSSION TITLE
Support for deleting saves

### DISCUSSION THREAD

**Discussion by nicolasnobelis** [mtkennerly/ludusavi-manifest#48] [General]
One feature I am missing coming from GameSave Manager is the possibility to delete the saves, for instance left after the uninstallation of a game.

Is it possible to do that with Ludusavi, or is it maybe on the roadmap?

Sorry if the subject was already answered, I could not find anything.

**Comment by mtkennerly** [comment]
> Sorry if the subject was already answered, I could not find anything.

No worries; it hasn't come up much. These are the only ones I remember: https://github.com/mtkennerly/ludusavi/issues/46, https://github.com/mtkennerly/ludusavi/issues/196

I have mixed feelings on this 😅 On one hand, it makes sense and there's nothing wrong with having the option if you use it responsibly. On the other hand, Steam has an option to delete your Proton prefix, and I see people post now and then about how they accidentally lost their saves because they were messing around with it. This may sound a bit silly, but I think having a deletion option in Ludusavi could give users a false sense of security that it's safe to delete files without doing some testing/research first. Maybe the game is still installed and they want to free up space, but the game doesn't handle the deletion properly, or the user doesn't realize that one of the files is configuration that shouldn't be deleted, or one of the files Ludusavi found is actually part of the game install data that was accidentally included in the PCGamingWiki save list.

As an alternative, you can quickly open the native file browser from the scan results and perform the deletion there. That also gives you a chance to check the context if there's anything else in the folder you want to delete or if you just want to get rid of the whole folder. For example, maybe you have a case like `steamapps/common/some-game/saves/user1/save1.dat`, where at best Ludusavi would let you delete `save1.dat`, but you'd still have an otherwise empty `some-game/saves` folder floating around.

> https://github.com/user-attachments/assets/2a1f10be-19d4-4e26-9713-72cb51a4bcfb

**Reply by nicolasnobelis** [reply]
Thanks for sharing you concerns.

My "workflow" is:

- I finish a game first (that I won't play anymore),
- I back up its saves using Ludusavi,
- I uninstall the game,
- I remove the saves to unclutter Ludusavi UI.

I have to uninstall the game before the last step because of sync features like Steam Cloud.

Regarding the user making mistakes, I think this could be addressed with a confirmation popup when the user press the button?

Anyway for the time being, I am doing like you say: deleting manually the saves using the file browser.

I am fine if the feature is not implemented yet. I just wanted to give you some arguments to tip the balance in favor of implementing it one day :smile:
